### PR TITLE
Display stack duration in profiler

### LINKED ISSUE #156: Display the stack duration in the profiler stack header
| Q            | A
| ------------ | ---
| Bug?         | no
| New Feature? | yes
| Version      | N/A

Display the stack duration in the profiler stack header.

### PR DESCRIPTION
**Pull request by fbourigault**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | fixes #156 
| Documentation   | N/A
| License         | MIT

This add the stack duration in the profiler.

httplugbundle4

### CODE REVIEW

File: Resources/public/style/httplug.css [hunk 1]
```diff
@@ -80,6 +80,11 @@
     font-size: 12px;
 }
 
+.httplug-duration {
+    min-width: 6ch;
```
Comment (Nyholm) [inline Resources/public/style/httplug.css]: "ch"? What is that? 

Someone once told me to always use "rem".

Comment (fbourigault) [inline Resources/public/style/httplug.css]: 1 ch is the width of a "0" in the current font. It looks easier to me as I want the box at least large as if it was filled with 6 characters.

Comment (Nyholm) [inline Resources/public/style/httplug.css]: hm. Interesting. Thanks. 

What if it takes more then one sec? "1000 ms" That is 7 chars, right? Will we get a line break? If so, change this to `7ch` and we will be all happy.

Comment (fbourigault) [inline Resources/public/style/httplug.css]: I use `number_format` twig filter, so you get a thousand separator. But anyway, I set this as a `min-width` to get the same width when duration is lower than 100ms and lower than 1000ms. When duration is over 1000ms, the box get larger.

Comment (Nyholm) [inline Resources/public/style/httplug.css]: Sorry. It was e reading this to fast. It is all good then =)

**Comment by Nyholm** [comment]
Thank you.

**Pull request merged**